packages/amplify-cli-core/src/index.ts

```diff
@@ -31,6 +31,7 @@ export * from './category-interfaces';
 export * from './customPoliciesUtils';
 export * from './utils/doc-links';
 export * from './utils/gql-transformer-version';
+export * from './spinner';
```

Comment:
I think it might make more sense if the spinner is apart of the `amplify-prompts` package instead of core. cc @edwardfoyle